app/mailers/producer_mailer.rb

```diff
@@ -92,6 +92,8 @@ def customer_data(line_items)
         quantity: line_item.quantity,
         first_name: order.billing_address.first_name,
         last_name: order.billing_address.last_name,
+        phone: order.billing_address.phone,
+        email: order.customer&.email,
```

Comment:
Part of me wonders if we can include these conditionally, only when `@display_customer_names`. 
But it probably makes for ugly code, and the condition in the view is nice and clear. As long as it's covered by specs..